### PR TITLE
docs: Wallet integration for phase-2 testnet

### PR DESCRIPTION
**Pull request by vitsalis**
Add documentation for wallet integration for the phase-2 testnet.

Pending elements:
* Wallet injectable interface for Babylon
* Staking back-end setup instructions

Also, these docs were written in a bit of a rush. They would benefit from some polishing and a graph or two.

### CODE REVIEW

File: bbn-test-5/integration/wallet/babylon-wallet.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+* Token minimum denomination: `ubbn` (6 decimals)
+* Human-readable denomination: `tbaby` (test baby)
+
+### Accounts, message signing, token balance, and token transfer
+
+The Babylon PoS chain utilises the default Cosmos SDK
+functionality for accounts, message signing,
+token balance, and token transfer.
+Please refer to the relevant Cosmos SDK documentation
+for more details.
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply bbn-test-5/integration/wallet/babylon-wallet.md:33]: @vitsalis wallet needs to export the
- `connect` method
- `getAddress`
- `getPublicKeyHex`
- `getBalance`
- `getOfflineSigner`

**Review by gbarkhatov — APPROVED** [review]

**Comment by gbarkhatov** [comment]
Looks good overall, though at the moment there is no mention of `bbn-wallet-connect`. But this lib does not have the docs for integration right now.

File: bbn-test-5/integration/wallet/babylon-wallet.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vitsalis) [reply bbn-test-5/integration/wallet/babylon-wallet.md:33]: This document is not intended for integrating the Babylon wallet into the staking dashboard, but for integrating the Babylon blockchain as a supported chain in the wallet.

This sentence aims to explain to the reader that in order to integrate the Babylon chain into their wallet, they should expect the default Cosmos SDK functionality for those features. It is up to them to decide how they are going to implement them to integrate the Babylon chain.

[the remark this answers is not in the document]
Comment (vitsalis) [reply bbn-test-5/integration/wallet/babylon-wallet.md:33]: The documentation of these required methods should be mentioned in the injectable wallet provider docs which are linked on the Bitcoin Staking dashboard integration docs.

**Comment by vitsalis** [comment]
> Looks good overall, though at the moment there is no mention of bbn-wallet-connect. But this lib does not have the docs for integration right now.

@gbarkhatov I was thinking we mention `bbn-wallet-connect` only for the wallet injectable interface that wallets should support in order to be integrated. When those docs are there we can update the link. We do not have to mention this for native wallet integration as we intend to maintain these integrations internally. Instead I point users to Tomo connect.

**Pull request merged**